### PR TITLE
Codecov

### PR DESCRIPTION
**Pull request by feliwir**
Added automatic code coverage

### CODE REVIEW

File: src/OpenSage.Game.Tests/OpenSage.Game.Tests.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>    
-    <TargetFramework>net462</TargetFramework>    
-    <RootNamespace>OpenSage.Tests</RootNamespace>
```
Comment (tgjones) [inline src/OpenSage.Game.Tests/OpenSage.Game.Tests.csproj]: Please add this `RootNamespace` back.

File: src/OpenSage.Game.Tests/OpenSage.Game.Tests.csproj [hunk 2]
```diff
@@ -14,16 +14,15 @@
...
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
+    <PackageReference Include="xunit" Version="2.4.0-beta.1.build3958" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0-beta.1.build3958" />
   </ItemGroup>
+  
   <ItemGroup>
     <ProjectReference Include="..\OpenSage.Game\OpenSage.Game.csproj" />
     <ProjectReference Include="..\OpenSage.Mods.BuiltIn\OpenSage.Mods.BuiltIn.csproj" />
   </ItemGroup>
-  <ItemGroup>
```
Comment (tgjones) [inline src/OpenSage.Game.Tests/OpenSage.Game.Tests.csproj]: Doesn't this get added by Visual Studio whenever you run tests locally?

Comment (feliwir) [reply src/OpenSage.Game.Tests/OpenSage.Game.Tests.csproj]: never saw it somewhere else. Checked the test projects of Eric and they didn't have it aswell

**Comment by tgjones** [comment]
Thank you! Now we should write some tests.

**Pull request merged**